docs/topics/multiplatform/multiplatform-dsl-reference.md

```diff
@@ -749,6 +763,68 @@ kotlin {
 </tab>
 </tabs>
 
+Alternatively, to configure compiler options that are common for all targets, you can use the `compilerOptions` [top-level block](multiplatform-dsl-reference.md#top-level-blocks):
+
+<tabs group="build-script">
+<tab title="Kotlin" group-key="kotlin">
+
+```kotlin
+kotlin {
+    jvm {
+        val main by compilations.getting {
+            compilerOptions.configure { 
+                // Set up the Kotlin compiler options for the 'main' compilation:
+                jvmTarget.set(JvmTarget.JVM_1_8)
+            }
+        
+            compileKotlinTask // get the Kotlin task 'compileKotlinJvm' 
+            output // get the main compilation output
+        }
+        
+        compilations["test"].runtimeDependencyFiles // get the test runtime classpath
+    }
```

Comment:
What do you think about skipping this configuration part in the example of alternative?